tests/EventEmitterTest.php

```diff
@@ -13,60 +13,62 @@
 
 use Evenement\EventEmitter;
 use InvalidArgumentException;
+use PHPUnit\Framework\Attributes\BackupGlobals;
+use PHPUnit\Framework\Attributes\BackupStaticProperties;
+use PHPUnit\Framework\Attributes\Before;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DoesNotPerformAssertions;
 use PHPUnit\Framework\TestCase;
 
+#[CoversClass(EventEmitter::class)]
+#[BackupGlobals(false)]
+#[BackupStaticProperties(false)]
```

Comment:
Note to self: Read up on these two attributes

Reply:
Honestly, I only added them as part of the change to using newer PHPUnit versions. This was based on the current `phpunit.xml.dist` within master having:

```xml
<phpunit backupGlobals="false"
         backupStaticAttributes="false"
        [...]
>
```

`BackupGlobals` specifies whether global and super-global variables should be backed up before a test and then restored after its run. While `BackupStaticProperties` is the same, except for static class properties.

I think we will be fine in dropping both as 1.) no static properties exist and 2.) the only global we rely on is a value we set in `tests/functions.php`. It does appear to have any adverse affect on the unit tests in removing those attributes.

Reply:
Removed in latest commit.